tracing/examples/sloggish/sloggish_subscriber.rs

```diff
@@ -278,8 +278,4 @@ impl Subscriber for SloggishSubscriber {
         // TODO: unify stack with current span
         self.current.exit();
     }
-
-    fn drop_span(&self, _id: tracing::Id) {
```

Comment:
Should this be replaced by a `TODO` marked `try_close`?

Reply:
Perhaps...this entire example could probably use a rewrite?